js/client.js

```diff
@@ -0,0 +1,280 @@
+//TODO: Consider using JS modules for imports
```

Comment:
I love this idea, if it ever feels natural to start doing this that would be awesome!